### PR TITLE
Import the canary component

### PR DESCRIPTION
**Pull request by alfakini**
Currently importing directly from https://github.com/KappaSigmaMu/canary-component

I had to add the `.obj` file in our project because Three.js requires an object url to load it. I don't know if it is possible to automatically import files from `node_modules`. Just found that people usually create scripts to explicitly copy the files they want.

Screen Shot 2021-07-18 at 15 48 04

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -30,15 +30,14 @@ jobs:
     - name: Upload coverage to Codecov
       uses: codecov/codecov-action@v1
 
-    - name: Run linter
-      run: yarn linter
```
Comment (murilooon) [inline .github/workflows/main.yml:34]: why?

File: package.json [hunk 2]
```diff
@@ -3,17 +3,76 @@
...
+    "terser-webpack-plugin": "4.2.3",
+    "ts-pnp": "1.2.0",
     "typescript": "^4.1.2",
-    "web-vitals": "^1.0.1"
+    "url-loader": "4.1.1",
+    "web-vitals": "^1.0.1",
+    "webpack": "4.44.2",
+    "webpack-dev-server": "3.11.1",
+    "webpack-manifest-plugin": "2.2.0",
+    "workbox-webpack-plugin": "5.1.4"
```
Comment (murilooon) [inline package.json:75]: Why did we add all these packages? Are they necessary?

File: package.json [hunk 3]
```diff
@@ -24,21 +83,22 @@
...
   },
   "scripts": {
-    "build": "react-scripts build",
+    "build": "node scripts/build.js",
     "ci": "yarn install --frozen-lockfile",
-    "eject": "react-scripts eject",
-    "start": "react-scripts start",
-    "test": "react-scripts test",
+    "start": "node scripts/start.js",
+    "test": "node scripts/test.js",
```
Comment (murilooon) [inline package.json:101]: Why node instead of react-scripts? Is needed for loading the canary?

**Review by danielSbastos — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (alfakini) [reply package.json:75]: From `yarn run eject` they were extracted from `create-react-app`.

File: package.json (hunk 3 above)
Comment (alfakini) [reply package.json:101]: From `yarn run eject`.

**Pull request merged**